var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -153,31 +161,31 @@ def configure_args(self):
         cppflags = []
         ldflags = []
         libs = []
-
-        config_args = [
-            "--enable-v2",
-            "--enable-utilities",
-            "--enable-static",
-            "--enable-largefile",
-            "--enable-netcdf-4",
-        ]
+        config_args = []
 
         if "+optimize" in self.spec:
             cflags.append("-O2")
 
         config_args.extend(self.enable_or_disable("fsync"))
+        config_args.extend(self.enable_or_disable("v2"))
+        config_args.extend(self.enable_or_disable("utilities"))
+        config_args.extend(self.enable_or_disable("largefile"))
+        config_args.extend(self.enable_or_disable("parallel-tests"))
+        config_args.extend(self.enable_or_disable("cdf5"))
+        config_args.extend(self.enable_or_disable("netcdf-4"))
+        config_args.extend(self.enable_or_disable("doxygen"))
```

Comment:
This does not work on my mac. Even though the default for `doxygen` is false and I don't see any place where it requested via `+doxygen`, spack's solver comes out with `+doxygen` in the spec, the configure line has `--enable-doxygen` and the error is:
```
Making all in docs
Making all in images
make[3]: Nothing to be done for `all'.
Doxyfile \
	cp auth.md obsolete/fan_utils.html html
make[3]: Doxyfile: No such file or directory
make[3]: *** [doxyfile.stamp] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
==> Error: ProcessError: Command exited with status 2:
    'make' '-j10' 'V=1'
```
Why do we need to build netcdf-c documentation at all? Clearly it's available on the web and we are not trying to develop netcdf-c within spack. Can we just remove the doxygen lines 176 and 86 altogether?
```